spec/amp-var-substitutions.md

```diff
@@ -252,6 +252,21 @@ For instance:
 <amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(cid-scope-cookie-fallback-name,user-consent-id)"></amp-pixel>
 ```
 
+### QUERY_STRING
```

Comment:
This would also be useful in amp-analytics. Do you mind updating https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md and https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/vendors.js? 

Thanks.